dev/NavigationView/NavigationViewItemPresenter.cpp

```diff
@@ -38,14 +38,16 @@ void NavigationViewItemPresenter::OnApplyTemplate()
             m_expandCollapseChevron.set(expandCollapseChevron);
             m_expandCollapseChevronTappedToken = expandCollapseChevron.Tapped({ navigationViewItem, &NavigationViewItem::OnExpandCollapseChevronTapped });
         }
-
         navigationViewItem->UpdateVisualStateNoTransition();
 
 
         // We probably switched displaymode, so restore width now, otherwise the next time we will restore is when the CompactPaneLength changes
-        if (navigationViewItem->GetNavigationView().PaneDisplayMode() != winrt::NavigationViewPaneDisplayMode::Top)
+        if(auto&& navigationView = navigationViewItem->GetNavigationView())
         {
-            UpdateCompactPaneLength(m_compactPaneLengthValue, true);
+            if (navigationViewItem->GetNavigationView().PaneDisplayMode() != winrt::NavigationViewPaneDisplayMode::Top)
```

Comment:
>navigationViewItem->GetNavigationView() [](start = 16, length = 39)

can use navigationView that was fetched above.

Reply:
Oh right, forgot that. #Fixed